feat(content): add mission about transporting a blind man from Martini (#8484)
endless-sky/endless-sky: data/human/human missions.txt

			choice
				`	"Hello."`
				`	"Are you alright?"`
				`	(Nevermind.)`
					decline
			`	"Yes, hello, how are you doing?" he says, shifting his head towards your direction. "I'm here waiting for someone to pick me up," he says in a voice that sounds like he expects it at any moment.`

			label questions
			choice
				`	"Are they far away?"`
					goto far
					to display
						not "Blind Man from Martini: label far"
				`	"Who is it you're looking for?"`
					goto looking
					to display
						has "Blind Man from Martini: label far"
				`	"Where is she?"`
					goto somewhere
					to display
						has "Blind Man from Martini: label looking"
			
			label far
			action
				set "Blind Man from Martini: label far"
			`	"Well, I suppose you could say that they are far away, or they could be right here. I don't know, I can't see them." He taps his cane twice more.`
				goto questions
			
			label looking
			action
				set "Blind Man from Martini: label looking"

> 			`	"Yes, I need to get to my daughter, Anita. Someone's meant to take me to her." Yet again, his cane taps twice.`

Author: Originally this was meant to be more interactive, and there were going to be more questions, but it was hard to write so many obscure answers. 

I'm wondering if I should make it more linear, but I'm going to keep the current structure right now.

			# The player can do three actions before arriving at the hospital.
			label taxi
			branch questions hospital
				"Blind Man from Martini: taxi questions count" != 3
			label questions
			choice
				`	"How did you become blind?"`
					goto blind
					to display
						not "Blind Man from Martini: label blind"

				`	"How did becoming blind change your life?"`
					goto blind-1
					to display
						has "Blind Man from Martini: label blind"
						not "Blind Man from Martini: label blind-1"

				`	"How do you feel about the atomic bombings?"`
					goto atomic-bombings
					to display
						has "Blind Man from Martini: label blind"
						not "Blind Man from Martini: label atomic-bombings"

				`	"Was your daughter caught in the atomic bombings?"`
					goto daughter
					to display
						or
							has "Blind Man from Martini: label blind"
							has "Blind Man from Martini: label illness"

> 						not "Blind Man from Martini: label daughter"

Reviewer: If you first ask him about his daughter as opposed to his blindness, your further option will include asking if she was caught in the nukes; but you don't necessarily know if _he_ was, either. It seems kind of an "off" question, if neither of you had mentioned nukes yet. 

I believe it should be further locked behind asking him about his blindness, going something like this: `"What happened to your daughter?"` -> `"How did you become blind?"`[only option] -> `"Was your daughter caught in the atomic bombings?"`, if you ask him about her first. ]

Also: 
```suggestion
						has "Blind Man from Martini: label blind"
						has "Blind Man from Martini: label illness"
						not "Blind Man from Martini: label daughter"
```
Those 2 above first lines seem to have an extra tab.

Author: The option is only meant to display if any of you mention nukes, which is why there's an "or" there and the lines are indented more. Maybe it doesn't work properly?

The option should not display if you've asked him already about it (label daughter)

Reviewer: I'd check again and with the changes, but sadly trying to download the updated artefact doesn't work at the moment, and neither does ESL2 (though the latter wasn't working earlier today, either). Not sure why, but i click to download and GitHub crashes. Weird! 

I'll get back to you once it works again, i suppose! And about the commits, great changes, and about the name, you're very much welcome :D

Reviewer: Alright, managed to redownload. And nope, it seems to not work properly! Here:
![image](https://user-images.githubusercontent.com/81255699/222838678-6abf8645-acbb-452d-9631-09370d4b63d9.png)

Secondly, when testing new paths i realised that this order was possible: 
![image](https://user-images.githubusercontent.com/81255699/222838328-0a21c731-1383-4d0b-8eee-416a8c1050df.png)
I believe that it would flow best if `what happened to your daughter` was locked out once you asked about his blindness, and only available again once you ask if she was caught in the bombing. Alternatively, you could lock out `Was your daughter caught in the atomic bombings` once he effectively answers she was not in the other question about her. Otherwise, it just seems a little off, in my opinion. It feels like the convo is the wrong way around if you "invert" that order, as he answers that her problems don't stem from the atomic bombs but from something on another planet! Either of those solutions would fix that.

Author: Mmm, I'll switch the two `daughter` labels around. Thanks.

					"Blind Man from Martini: taxi questions count" == 1
				goto taxi

			`	As the taxi drives on a highway on the side of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers.`
				to display
					"Blind Man from Martini: taxi questions count" == 2
				goto taxi

			`	The taxi enters the University district proper now, and the window shows thongs of university students from the University of New Sydney travelling to and from buildings, going over steel walkways and on footpaths beside the streets, clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney; the University of Chiron's campus is on the other side of town. Presumably, to keep the fierce competition between the two institutions from boiling over each other.`
				to display
					"Blind Man from Martini: taxi questions count" == 3
				goto taxi

			label hospital

> 			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be more easily treated in a place bright and alive than cold and white.`

Reviewer: ```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people could be more easily treated in a place bright and alive than cold and white.`
```
or
```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be treated in a place bright and alive instead of cold and white.`
```
or
```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be treated in a place bright and alive instead of cold and white.`
```
All work better, i believe!

			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
			choice
				`	"We're looking for Anita Pears, where can we find her?"`
			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
			`	"That's me," Douglas interjects before you can speak.`

> 			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Douglas is booked in for today; we have a family-only policy, to keep the privacy of our patients."`

Reviewer: ```suggestion
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for today; we have a family-only policy, to keep the privacy of our patients."`
```
Calling him by his first name does not sound professional enough of a seemingly well trained receptionist, so calling him mister fits better, i believe.

			

event "quicksilver mixup 2 available"

> # Rationale for Douglas' actions; in 1000 years, treatment for blindness would be expected to be better. So he is able to do some limited actions, like recognize people, but not much more.

Reviewer 2: Was there any specific reason to leave this in? I couldn't think of any case where this popped up.

Author: I'm not so sure now, I can delete it.

Reviewer 3: "Blind" doesn't always mean "can't see anything at all." Many blind people do actually have some light sensitivity. You also don't necessarily need to be able to see to recognise a person.

Reviewer 2: ```suggestion
```

mission "Blind Man from Martini"
	name "Transport a blind passenger to <planet>"
	description "This blind man, Douglas, wishes to be transported to <destination>, so he can be with his daughter while she undergoes treatment there. He will pay you <payment>."
	source Martini
	destination Chiron
	to offer
		has "main plot completed"
		random < 15
	on offer
		conversation

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`

Author: Recently I had a PR that updated Martini's spaceport. It's destroyed in the nuclear bombings, then another existing spaceport is designated the planetary spaceport but that's too small to handle the traffic.

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`

Author: ```suggestion
			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He sits on a bench with a suitcase at his side, looking at and doing nothing particular aside from the occasional tap of his cane when someone passes by.`
```

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`

Reviewer 3: #7854
~~Though I don't necessarily agree that the spaceport should be destroyed.~~

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`

Author: I'd like to hear why, but probably better on Discord than this PR.

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He's just sitting on a bench looking ahead at nowhere in particular, with a suitcase to his side, and holding a cane in one hand. Nobody's paying any attention to him, but he is tapping his cane every so often someone passes by him.`

Reviewer 2: Well, if we revert the change, we can just make the change here as well.

			choice
				`	(Go up to him.)`
				`	(Ignore him.)`
					decline

> 			`	Wading through the mix of passengers, stevedores and captains in the overflowing corridor, you reach the man with the cane. It's obvious now that he's blind; he doesn't move to look at you and continues looking straight ahead, but he taps his cane twice when you come by him.`

Author: Maybe, but why should I say he's blind before that?

> 			`	Wading through the mix of passengers, stevedores and captains in the overflowing corridor, you reach the man with the cane. It's obvious now that he's blind, as he doesn't move to look at you and continues looking straight ahead, but he taps his cane twice when you come by him.`

Author: ```suggestion
			`	Wading through the mix of passengers, stevedores and captains in the overflowing corridor, you reach the man with the cane. It's obvious now that he's blind; he doesn't move to look at you and continues looking straight ahead, but he taps his cane twice when you come by him.`
```

				`	"Where is she?"`
					goto somewhere
					to display
						has "Blind Man from Martini: label looking"
				`	(Nevermind, he's just a blind man.)`
					decline
			
			label far
			action
				set "Blind Man from Martini: label far"
			`	"Well, I suppose you could say that they are far away, or they could be right here. I don't know, I can't see them." He taps his cane twice more.`
				goto questions
			
			label looking
			action
				set "Blind Man from Martini: label looking"
			`	"Someone to take me to my daughter, Anita." Yet again, his cane taps twice.`
				goto questions

			label somewhere
			`	He looks over to his side, as if he feels reserved about something. "She's on <planet>, getting some experimental treatment from the University of New Sydney. I want to be by her side if anything goes wrong, to tell her that I'm there for her, that I love her, and that everything will be okay. She's been waiting for the treatment for so long now. And since nobody else seems to have arrived here yet, I hope you'll be the one to take me there."`
			choice
				`	"Of course, I'll take you to <planet>."`
				`	"Sorry, I won't be able to."`
					goto can't

> 			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas, and he promises to you <payment> for transporting him.`

Reviewer 2: ```suggestion
			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas Pears, and he promises to you <payment> for transporting him.`
```
Need the last name because it isn't mentioned before `<first>` mentions it at the hospital. I'd suggest either adding an option for the player to ask about payment before agreeing (or just have Douglas mention it before the player decides).

				accept

			label can't
			`	"Then I will wait until another comes to help me." The blind man taps his cane once more, then returns to looking straight ahead.`
				decline

	on complete
		log `Brought a blind man named Douglas to Chiron so he could visit his daughter, who will undergo experimental treatment at the University of New Sydney.`
		payment 58000
		conversation

> 			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. He guides both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`

Author: Well, after the player's landed on Chiron, it's possible they've forgotten who Douglas is (honestly I'd do that) especially if they come from Martini and then go somewhere else before coming back to Chiron.  

I only mention blindness 8 times in the dialogue, though.

`Intercity` is correct, so I guess `Intracity` also is. I made sure to check though. https://www.merriam-webster.com/dictionary/intracity

> 			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. They guide both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`

Author: I just hadn't decided, to be honest.

> 			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. They guide both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`

Author: ```suggestion
			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. He guides both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`
```

			`	"Well, I must go now to the University of New Sydney. They and my daughter are expecting me at their Health Sciences quarter."`
			choice
				`	"Goodbye."`
					goto bye
				`	"I hope your daughter has a safe recovery."`
					goto recovery
				`	"Everything is going to be alright."`
					goto recovery
				`	"Could I come with you?"`
					goto comealong
			label bye

> 			`	"Bye, now. I wish you greater luck in your travels and your sights. Take it from me, a man who cannot walk or see by himself, without aid; there is no greater sight than to see." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going towards University district on the far side of town.`

Author: 'University' is the name of the district, but I could make it more clear.

> 			`	"Bye, now. I wish you greater luck in your travels and your sights. Take it from me, a man who cannot walk or see by himself, without aid; there is no greater sight than to see." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going towards University district on the far side of town.`

Author: ```suggestion
			`	"Bye, now. I wish you greater luck in your travels and your sights. Take it from me: there is no greater sight than to see." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going toward the University of Chiron campus on the far side of town.`
```

				decline

			label recovery
			`	"Yes, thank you so much. I hope so too, of course; actually, it's a little foolish saying that, of course I do. But now I have to go. Again, thank you for making this journey possible."`
			choice
				`	"Goodbye."`
					goto bye
				`	"Could I come with you?"`
					goto comealong
			
			label comealong

> 			`	Douglas looks taken aback for a moment, but says, "Well, I suppose you can come along. I'm not sure if the university would let you in, though. But I think Anita would like to meet you, or at least say hello." You get into the taxi with him, and it starts toward New Sydney's University district.`

Reviewer 3: ```suggestion
			`	Douglas looks taken aback for a moment, but says, "Well, I suppose you can come along. I'm not sure if the university would let you in, though. But I think Anita would like to meet you, or at least say hello." You get into the taxi with him, and it starts toward New Sydney's university district.`
```
Is this used as a proper noun?

Author: Yes, University is a proper noun.

				goto taxi

			label blind-1
			action
				set "Blind Man from Martini: label blind-1"
				"Blind Man from Martini: taxi questions count" ++
			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. I took my life for granted then, but anything can be taken away from you if you get too used to it."`
				goto taxi
			
			label atomic-bombings
			action
				set "Blind Man from Martini: label atomic-bombings"
				"Blind Man from Martini: taxi questions count" ++
			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`

> 			`	"And for the bombings themselves, they were certainly a malicious act. The Syndicate needs to take full accountability for it, not only for the people who were killed or affected in some form or another. I'm too cynical to say that they will, with all the power they still have in Parliament."`

Reviewer 3: Does checkmate actually expose the Syndicate's involvement?

Author: No, but I can use branch to change the dialogue.

				goto taxi

			label waiting
			action 
				"Blind Man from Martini: taxi questions count" ++
			`	Deciding not to ask Douglas anything, you look out the windows of the taxi. You're in the midst of New Sydney, and skyscrapers stare down at your tiny taxi from each side of the avenue, but despite this it is still quite green and cheery, with trees on each side and people walking to work in the morning.`
				to display
					"Blind Man from Martini: taxi questions count" == 1
				goto taxi

			`	As the taxi drives on a highway on the edge of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers. Far away on the horizon, you can barely notice the smokestacks of factories.`
				to display
					"Blind Man from Martini: taxi questions count" == 2
				goto taxi

> 			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney; the University of Chiron's campus is on the other side of town. Presumably, to keep the fierce competition between the two institutions from boiling over each other.`

Reviewer 2: ```suggestion
			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that this isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
```

Author: ```suggestion
			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
```

				to display
					"Blind Man from Martini: taxi questions count" == 3
				goto taxi

			label hospital

> 			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside, to match the architecture of the surroundings, and strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative and that people should be treated in a place bright and alive instead of cold and white.`

Reviewer 2: ```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
```
Seems a bit too... "editorialized" for a narrator, if you will (though I totally agree :)

			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
			choice

> 				`	"We're looking for Anita Pears, where can we find her?"`

Reviewer 2: ```suggestion
				`	"We're looking for Anita Pears. Where can we find her?"`
```

			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
			`	"That's me," Douglas interjects before you can speak.`

> 			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for today; we have a family-only policy, to keep the privacy of our patients."`

Reviewer 2: ```suggestion
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for the his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
```
`for today` makes me think, "oh, this procedure is day-specific. Maybe this mission should have a deadline?" I think rewording this is a better fix than adding a deadline.

Author: ```suggestion
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
```

			choice
				`	"All right."`
					goto end
				`	"Are you sure?"`
			`	"We can't allow immediate exceptions, sorry. There's nothing I can do."`
			label end
			`	Douglas gives you a compassionate look, and says to the receptionist, "There'll be no worry." He turns back to you once more and says, "I'm sure that when Anita is recovered and well, we can meet again some day. I don't want to keep you from your business too long, either. I expect ship captains are busy folk." He comes up to you and, to your surprise, gives you a hug. "Thank you so much for bringing me here and coming along with me. I really can't say how I want to express my gratitude. But I hope we meet again. Farewell."`
			choice
				`	"Goodbye."`
				`	"Farewell to you too."`

> 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant towards his daughter's room, leaving you in the reception.`

Reviewer 2: ```suggestion
			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
```

				`	"Where is she?"`
					goto somewhere
					to display
						has "Blind Man from Martini: label looking"
				`	"I see. Safe travels to you." (Walk away.)`
					decline
			
			label far
			action
				set "Blind Man from Martini: label far"
			`	"Well, I suppose you could say that they are far away, or they could be right here. I don't know, I can't see them." He taps his cane twice more.`
				goto questions
			
			label looking
			action
				set "Blind Man from Martini: label looking"
			`	"Someone to take me to my daughter, Anita." Yet again, his cane taps twice.`
				goto questions

			label somewhere
			`	He looks over to his side, as if he feels reserved about something. "She's on <planet>, getting some experimental treatment from the University of New Sydney. I want to be by her side if anything goes wrong, to tell her that I'm there for her, that I love her, and that everything will be okay. She's been waiting for the treatment for so long now. And since nobody else seems to have arrived here yet, I hope you'll be the one to take me there."`
			choice
				`	"Of course, I'll take you to <planet>."`
				`	"Sorry, I won't be able to."`
					goto can't

> 			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas Pears, and he promises to you <payment> for transporting him.`

Reviewer 2: ```suggestion
			`	The man musters a broad smile, says "Thank you all too much. Would you like to help me with the luggage?" He scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas Pears, and he promises to you <payment> for transporting him.`
```

				accept

			label can't
			`	"Then I will wait until another comes to help me." The blind man taps his cane once more, then returns to looking straight ahead.`
				decline

	on complete
		log `Brought a blind man named Douglas to Chiron so he could visit his daughter, who will undergo experimental treatment at the University of New Sydney.`
		payment 58000
		conversation

> 			`Throughout the trip, Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. He guides both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while Douglas arranges for the <payment> to be sent to you.`

Reviewer 2: ```suggestion
			`Throughout the trip, Douglas has been relatively quiet and has kept to himself. When you land, he lets you help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. He guides both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while Douglas arranges for the <payment> to be sent to you.`
```

			`	"Well, I must go now to the University of New Sydney. They and my daughter are expecting me at their Health Sciences quarter."`
			choice
				`	"Goodbye."`
					goto bye
				`	"I hope your daughter has a safe recovery."`
					goto recovery
				`	"Everything is going to be alright."`
					goto recovery
				`	"Could I come with you?"`
					goto comealong
			label bye

> 			`	"Bye, now. I wish you greater luck in your travels and your sights. Take it from me: there is no greater sight than to see." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going toward the University of New Sydney campus on the far side of town.`

Reviewer 2: ```suggestion
			`	"Bye, now. I wish you greater luck in your travels and your sights." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going toward the University of New Sydney campus on the far side of town.`
```

				"Blind Man from Martini: taxi questions count" ++
			`	"It was the fourth of July, 3014. I was in my office, on one of the high rises on the outskirts of the capital, with my office window pointed right at the city center." He turns to look out the window, as if he knew New Sydney's skyline was out there where he couldn't see. In the small lull, he takes an audible breath.`
			`	"It was a beautiful day that day. I... I remember that Pollux was especially bright and colorful that day, and the songbirds that inhabit the city were alive with sound. They were unusually so. It was very vivid, very strange. Maybe I should have listened to the birds...`
			`	"But when the bomb dropped, I was looking right at it, from my office window. All I could remember next was that I was in a lot of pain, that I couldn't see, but my secretary - thank goodness for her - found me afterwards, during the evacuation, covered in glass from the shockwave. I survived mostly because I was sitting at my terminal and that protected me from the bulk of the shards.`
				goto taxi

			label blind-1
			action
				set "Blind Man from Martini: label blind-1"
				"Blind Man from Martini: taxi questions count" ++

> 			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. I took my life for granted then, but anything can be taken away from you if you get too used to it."`

Reviewer 2: ```suggestion
			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. Anything can be taken away from you if you get too used to it."`
```

				goto taxi
			
			label atomic-bombings
			action
				set "Blind Man from Martini: label atomic-bombings"
				"Blind Man from Martini: taxi questions count" ++

> 			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`

Reviewer 3: Did the treatment he got actually do anything?

Author: I was thinking that he could live by himself fairly well, instead of needing a caretaker or a guide dog or someone like that. But if the treatment's effects should be spelled out, I'm happy to find something or hear suggestions. The main point was that people were affected by the bombings, and I read that permanent blindness from the flash of the explosion was possible.

The mission was partly inspired by a documentary on Al Jazeera about a Polish man who had incurable blindness; during filing he could still barely see, but his eyesight was getting worse and he relied on his son. The name of the documentary is a little ironic for ES: "A Sky Without Stars". 

While the source of Douglas' blindness is different to the Polish man's, I'd think that in ES's universe the both Douglas and the Polish man's blindness could be cured. What the treatment means, I don't really know.

Reviewer 2: ```suggestion
			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. In my case, I wanted treatment and received it. But others, no...`
```

Author: I really appreciate that you took the time to ask someone about this! I would commit this suggestion (I understand the change), but the purpose of that line was to highlight that there are other people blinded by the explosion who could not get treatment because of the culture in the Paradise Worlds. For example, there's talk of servants and people that are indentured servants in all but name in the Paradise Worlds. I don't imagine that they would be allowed treatment because they couldn't afford it, or because of the breaking down of the health system due to the bombings. Douglas becomes aware of this because he's suffered something similar, but as he's in a better position than them he can get a faster or better treatment; he's grown to realise that he was probably too selfish then in jumping the line. Just some thoughts.

Reviewer 2: Happy to do it, and appreciate the openness to suggestions! I think that's a fair point. The person I spoke to discussed the idea of saying he's "fortunate" as a way of negatively viewing blindness, which could be rude to people who were born that way. Since Douglas isn't born blind and instead becomes blind after experiencing sight, that's a bit different (and this is where the person I consulted was a little unsure herself). So up to you, I'd say. I think there's arguments for both lines on this, so a bit of a toss-up.

Reviewer 3: He talks about needing help to "just live," and it seems to me like he's speaking from personal experience (that is to say, *he* himself needed that help), so it seems a little strange that he goes on to talk about treatment he was able to get, because he doesn't actually give any indication of how that treatment helped, what difference it made, and why others may have had a harder time without it.

Orientation and mobility training is a real life thing that may be relevant: helps to provide blind/visually impaired people with skills to aid them in navigating without sight. For example, cane training.
There's also vocational rehabilitation, which helps provide other skills. In general, it can help provide blind/VI people with the sorts of skills needed to "just live."

Having or lacking access to these sorts of things would certainly make a huge difference for people in Douglas' situation.

Author: Ok, this is a good question. I might think about adding some of this in another PR now that it's already been merged, maybe with roadrunner's suggestion too.

			# This will need to be updated once new Human campaigns are written.
			branch reconciliation checkmate
				has "free worlds reconciliation"
			label reconciliation
			`	"And for the bombings themselves, they were certainly a malicious act. The Syndicate needs to take full accountability for it, not only for the people who were killed or affected in some form or another. I'm too cynical to say that they will, with all the power they still have in Parliament."`
				goto taxi

			label checkmate

> 			`	"And for the bombings themselves, they were certainly a malicious act. I hear the authorities are still investigating the bombings, although it's very curious that Korban, the CEO of the Syndicate, resigned as soon as the Republic starting poking their heads that way. I'm still not sure who's responsible for the bombings, the deaths, the war... I can only hope the investigation will be thorough."`

Reviewer 3: ```suggestion
			`	"And for the bombings themselves, they were certainly a malicious act. I hear the authorities are still investigating the bombings, although it's very curious that Korban, the CEO of the Syndicate, resigned as soon as the Republic started poking their heads that way. I'm still not sure who's responsible for the bombings, the deaths, the war... I can only hope the investigation will be thorough."`
```

				to display
					"Blind Man from Martini: taxi questions count" == 3
				goto taxi

			label hospital
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
			choice
				`	"We're looking for Anita Pears. Where can we find her?"`

> 			`	For a moment, the receptionist shuffles information arounds on her terminal. "You are her father, Douglas Pears?"`
> 			`	"That's me," Douglas interjects before you can speak.`

Reviewer 2: ```suggestion
			`	For a moment, the receptionist shuffles information around on her terminal. "Are one of you a parent?"`
			`	"I'm the father, Douglas Pears," he replies.`
```
If the receptionist is addressing `<first>` here, asking if they're the father assumes a gender.

			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
			choice
				`	"All right."`
					goto end
				`	"Are you sure?"`
			`	"We can't allow immediate exceptions, sorry. There's nothing I can do."`
			label end

> 			`	Douglas gives you a compassionate look, and says to the receptionist, "There'll be no worry." He turns back to you once more and says, "I'm sure that when Anita is recovered and well, we can meet again some day. I don't want to keep you from your business too long, either. I expect ship captains are busy folk." He comes up to you and, to your surprise, gives you a hug. "Thank you so much for bringing me here and coming along with me. I really can't say how I want to express my gratitude. But I hope we meet again. Farewell."`

Reviewer 2: Is the thought behind the extra gratitude ("Thank you _so much_") because he's blind or because he's just that appreciative outside of the blindness? If the former, let's remove the extra thankfulness because it makes Douglas seem as though he would've been helpless without `<first>`'s help.

Author: I hope it's the second one (he has grown after the experience of the bombings to be more appreciative), but if it comes across like the first one I will remove it.

Reviewer 2: Probably fine to keep it as-is, then. Just wanted to confirm!

		fleet "Small Syndicate"
		to spawn
			has "Argosy Hijacking: declined"

mission "Blind Man from Martini"
	name "Transport a blind passenger to <planet>"
	description "This blind man, Douglas, wishes to be transported to <destination>, so he can be with his daughter while she undergoes treatment there. He will pay you <payment>."
	source Martini
	destination Chiron
	to offer
		has "main plot completed"
		random < 15
	on offer
		conversation

> 			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He sits on a bench with a suitcase at his side, looking at and doing nothing particular aside from the occasional tap of his cane when someone passes by.`

Author: ```suggestion
			`As you walk through <origin>'s cramped new planetary spaceport, you see a middle-aged man on the other side of the corridor, behind a column of people. He sits on a bench with a suitcase at his side, doing nothing particular aside from the occasional tap of his cane when someone passes by.`
```

			`	"Well, I must go now to the University of New Sydney. They and my daughter are expecting me at their Health Sciences quarter."`
			choice
				`	"Goodbye."`
					goto bye
				`	"I hope your daughter has a safe recovery."`
					goto recovery
				`	"Everything is going to be alright."`
					goto recovery
				`	"Could I come with you?"`
					goto comealong
			label bye
			`	"Bye, now. I wish you greater luck in your travels and your sights." With that, he taps his cane twice, and then steps into one of New Sydney's taxis, going toward the University of New Sydney campus on the far side of town.`
				decline

			label recovery
			`	"Yes, thank you so much. I hope so too, of course; actually, it's a little foolish saying that, of course I do. But now I have to go. Again, thank you for making this journey possible."`
			choice
				`	"Goodbye."`
					goto bye
				`	"Could I come with you?"`
					goto comealong
			
			label comealong

> 			`	Douglas looks taken aback for a moment, but says, "Well, I suppose you can come along. I'm not sure if the university would let you in, though. But I think Anita would like to meet you, or at least say hello." You get into the taxi with him, and it starts toward New Sydney's University district.`

Author: ```suggestion
			`	Douglas looks taken aback for a moment, but says, "Well, I suppose you can come along. I'm not sure if the university would let you in, though. But I think Anita would like to meet you, or at least say hello." You get into the taxi with him, and it starts toward New Sydney's University District.`
```

					to display
						or
							has "Blind Man from Martini: label blind"
							has "Blind Man from Martini: label illness"
						not "Blind Man from Martini: label daughter"
				
				`	"What happened to your daughter?"`
					goto illness
					to display
						not "Blind Man from Martini: label illness"
						has "Blind Man from Martini: label daughter"

				`	(Stay silent.)`
					goto waiting
					to display
						not "Blind Man from Martini: label waiting"

			label blind
			action
				set "Blind Man from Martini: label blind"
				"Blind Man from Martini: taxi questions count" ++
			`	"It was the fourth of July, 3014. I was in my office, on one of the high rises on the outskirts of the capital, with my office window pointed right at the city center." He turns to look out the window, as if he knew New Sydney's skyline was out there where he couldn't see. In the small lull, he takes an audible breath.`
			`	"It was a beautiful day that day. I... I remember that Pollux was especially bright and colorful that day, and the songbirds that inhabit the city were alive with sound. They were unusually so. It was very vivid, very strange. Maybe I should have listened to the birds...`

> 			`	"But when the bomb dropped, I was looking right at it, from my office window. All I could remember next was that I was in a lot of pain, that I couldn't see, but one of my colleagues - thank goodness for her - found me afterwards, during the evacuation, covered in glass from the shockwave. I survived mostly because I was sitting at my terminal and that protected me from the bulk of the shards."`

Author: Realism : ) I can't give a good big response right now, but I have a couple thoughts.

> And a 1-megaton nuke is HUGE. So 21km is the upper limit of how far he could have been from the explosion.

> Within a 6-km (3.7-mile) radius of a 1-megaton bomb, blast waves will produce 180 tonnes of force on the walls of all two-story buildings, and wind speeds of 255 km/h (158 mph).

Perhaps it's not impossible for the nuke to have been more than 1 megaton. US warheads today are something like 400 kilotons (https://en.wikipedia.org/wiki/W88), and up to 12 of them fit on one Trident II missile (https://en.wikipedia.org/wiki/UGM-133_Trident_II).

The Syndicate could have made one huge nuclear missile or a lot of small nuclear missiles. That could mean that they might have not only placed a nuke in the centre of the city, but also ones maybe closer to the outskirts, which still could have hit highrises there.

> And the radiation treatment meds must also be extremely effective if he survived in such a good shape.

Well, if nothing else, 3000 AD humanity has to show for something...

				goto taxi

			label blind-1
			action
				set "Blind Man from Martini: label blind-1"
				"Blind Man from Martini: taxi questions count" ++
			`	"Hmm... well, it was really only one of many things that hit my life then, not least of which was my newfound blindness, and the loss of my company and my job." He gives a heavy sigh, and he shifts in his seat as if something heavy was placed on his shoulders. "I was able to keep things intact for a while, but no, it was too much. Anything can be taken away from you if you get too used to it."`
				goto taxi
			
			label atomic-bombings
			action
				set "Blind Man from Martini: label atomic-bombings"
				"Blind Man from Martini: taxi questions count" ++

> 			`	"How do I feel about the bombings? Well, they changed everything. In some ways I'm very grateful that it showed - no, it exposed to me, that people, they- there's people that live this way. With no sight. Who need other people to help them just... live. And that there are plenty of people who don't have people to help them live. I was lucky, I could get treatment for my blindness. But others, no...`

Reviewer 4: Are the ` - ` and `- ` spaced differently on purpose?

Author: One is used like a semicolon (hopefully correctly), another is a change in the middle of dialogue. I could change the first dash to a semicolon.

Reviewer 4: I think it's fine as-is, just wanted to make sure.

				goto taxi

			label illness
			action
				set "Blind Man from Martini: label illness"
				"Blind Man from Martini: taxi questions count" ++
			`	"I don't know if I really should tell you this, but Anita was working on a project on nanomachines at CyberAcme when she got back. She was, uh... infected by them by accident. The experimental treatment is meant to try and eliminate the rogue machines, but since they're so small... that's why the treatment is experimental."`
				goto taxi

			label waiting
			action
				"Blind Man from Martini: taxi questions count" ++
			`	Deciding not to ask Douglas anything, you look out the windows of the taxi. You're in the midst of New Sydney, and skyscrapers stare down at your tiny taxi from each side of the avenue, but despite this it is still quite green and cheery, with trees on each side and people walking to work in the morning.`
				to display
					"Blind Man from Martini: taxi questions count" == 1
				goto taxi

> 			`	As the taxi drives on a highway on the edge of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers. Far away on the horizon, you can barely notice the smokestacks of factories.`

Reviewer 4: That district again.

Author: ```suggestion
			`	As the taxi drives on a highway on the edge of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial sector where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University District, a series of clean and white, flowing buildings and skyscrapers. Far away on the horizon, you can barely notice the smokestacks of factories.`
```

				to display
					"Blind Man from Martini: taxi questions count" == 2
				goto taxi

			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
				to display
					"Blind Man from Martini: taxi questions count" == 3
				goto taxi

			label hospital

> 			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`

Reviewer 4: ```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University District. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
```

Okay, so based on that earlier paragraph, there are multiple university districts but only one University District. I guess that should be capitalized instead of lowercase, then. Double-check the previous instances, too.

			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
			choice
				`	"We're looking for Anita Pears. Where can we find her?"`
			`	For a moment, the receptionist shuffles information around on her terminal. "Are one of you a parent?"`
			`	"I'm the father, Douglas Pears," he replies.`
			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
			choice
				`	"Alright."`
					goto end
				`	"Are you sure?"`
			`	"We can't allow immediate exceptions, sorry. There's nothing I can do."`
			label end

> 			`	Douglas gives you a compassionate look, and says to the receptionist, "There'll be no worry." He turns back to you once more and says, "I'm sure that when Anita is recovered and well, we can meet again some day. I don't want to keep you from your business too long, either. I expect ship captains are busy folk." He comes up to you and, to your surprise, gives you a hug. "Thank you so much for bringing me here and coming along with me. I really can't say how I want to express my gratitude. But I hope we meet again. Farewell."`

Reviewer 4: Maybe a reunion mission / thank you letter could be added.

Author: Yeah, I had thought about writing more - especially in the hospital - but I didn't because the mission was already pretty heavy, if you know what I mean, and there would be a lot of similarity to the Riley mission. It wouldn't be too much work to add a "thank you" dialogue, I guess.

			choice
				`	"Goodbye."`
				`	"Farewell to you too."`

> 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`

Author: ```suggestion
			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
			action
				clear "Blind Man from Martini: label blind"
				clear "Blind Man from Martini: label blind-1"
				clear "Blind Man from Martini: label atomic-bombings"
				clear "Blind Man from Martini: label illness"
				clear "Blind Man from Martini: label daughter"
				clear "Blind Man from Martini: label waiting"
				clear "Blind Man from Martini: taxi questions count"
```

> 				decline

Reviewer 4: I think you can unset all those conditions and stuff you used in the mission to clear up the save file a bit.

Author: I was following off how the Hai Revealed missions did these so I hadn't thought of this yet. That would be a good idea.

			choice
				`	"Goodbye."`
				`	"Farewell to you too."`
			action
				clear "Blind Man from Martini: label blind"
				clear "Blind Man from Martini: label blind-1"
				clear "Blind Man from Martini: label atomic-bombings"
				clear "Blind Man from Martini: label illness"
				clear "Blind Man from Martini: label daughter"
				clear "Blind Man from Martini: label waiting"
				clear "Blind Man from Martini: taxi questions count"
				clear "Blind Man from Martini: label far"
				clear "Blind Man from Martini: label looking"
			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`

> 			decline

Reviewer 3: ```suggestion
				decline
```
Or just remove the line entirely.
```suggestion
```
